int32 id
float64 local_x
float64 local_y
string type
string status